cf/test/test_DimensionCoordinate.py

```diff
@@ -484,14 +484,118 @@ def test_DimensionCoordinate_set_bounds(self):
 
         self.assertTrue((c.bounds.data.array == [0, 2592000]).all())
 
-    def test_DimensiconCoordinate__getitem__(self):
+    def test_DimensionCoordinate__getitem__(self):
         dim = self.dim
         self.assertTrue((dim[1:3].array == dim.array[1:3]).all())
 
         # Indices result in a subspaced shape that has a size 0 axis
         with self.assertRaises(IndexError):
             dim[[False] * dim.size]
 
+    def test_DimensionCoordinate_create_bounds(self):
+        # Create bounds for Voronoi cells
+        d = cf.DimensionCoordinate(data=[0.0, 40, 90, 180])
+        b = d.create_bounds()
+        self.assertTrue(
+            (
+                b.array
+                == np.array(
+                    [
+                        [-20.0, 20.0],
+                        [20.0, 65.0],
+                        [65.0, 135.0],
+                        [135.0, 225.0],
+                    ]
+                )
+            ).all()
+        )
+
+        d = d[::-1]
+        b = d.create_bounds()
+        self.assertTrue(
+            (
+                b.array
+                == np.array(
+                    [
+                        [225.0, 135.0],
+                        [135.0, 65.0],
+                        [65.0, 20.0],
+                        [20.0, -20.0],
+                    ]
+                )
+            ).all()
+        )
+
+        # Cyclic coordinates have a co-located first and last bound:
+        d.period(360)
+        d.cyclic(0)
+        b = d.create_bounds()
+        self.assertTrue(
+            (
+                b.array
+                == np.array(
+                    [
+                        [270.0, 135.0],
+                        [135.0, 65.0],
+                        [65.0, 20.0],
+                        [20.0, -90.0],
+                    ]
+                )
+            ).all()
+        )
+
+        # Cellsize units must be equivalent to the coordinate units,
+        # or if the cellhas has no units then they are assumed to be
+        # the same the coordinates:
```

Comment:
```suggestion
        # Cellsize units must be equivalent to the coordinate units,
        # or if the cell has no units then they are assumed to be
        # the same as the coordinates:
```

(equivalent to the changes made in the docstring, if that saves you work checking...)